repo-projects/msbuild.proj

```diff
@@ -1,7 +1,8 @@
 <Project Sdk="Microsoft.Build.NoTargets">
 
   <PropertyGroup>
-    <BuildArgs>$(BuildArgs) $(FlagParameterPrefix)warnAsError $(ArcadeFalseBoolBuildArg)</BuildArgs>
+    <!-- MSBuild uses a too old version that creates a torn state when being used with a >= .NET 10 P4 SDK. -->
```

Comment:
that's interesting we should be updating it weekly and be very close to the same version . . .